File: eat-it/src/main/java/ide/eatit/model/dto/RecipeDto.java
package ide.eatit.model.dto;

import ide.eatit.model.User;

public class RecipeDto {
    private Integer id;

    private String name;

    private String description;

    private String createdBy;

    public RecipeDto(Integer id, String name, String description, String createdBy) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.createdBy = createdBy;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getCreatedBy() {
        return createdBy;
    }

    public void setCreatedBy(String createdBy) {
        this.createdBy = createdBy;
    }

}

File: eat-it-fe/src/app/models/dtos/recipe-dto.ts
export class RecipeDto {
    id: number;
    name: string;
    description: string;
    createdBy: string;
  
    constructor(id: number, name: string, description: string, createdBy: string) {
      this.id = id;
      this.name = name;
      this.description = description;
      this.createdBy = createdBy;
    }
  }
  